2200070/소득불균형.py

```diff
@@ -0,0 +1,16 @@
+t=int(input())
+
+for num in range(1, t+1):
+    n=int(input())
+    result=0
+    p=list(map(int, input().split()))
+    a=int(sum(p) / len(p))
```

Comment:
깔끔한 풀이군요 👍 
```suggestion
    a = sum(p) / len(p)
```
정확한 평균값은 `int` 없이 구해주는 게 맞습니다! (이 문제에서는 정답에 영향을 끼치지 않습니다)